types/knockout/index.d.ts

```diff
@@ -217,6 +218,7 @@ interface KnockoutVirtualElements {
 }
 
 interface KnockoutExtenders {
+    [userland: string]: (target: KnockoutObservable<{}>, ...args) => any;
```

Comment:
it is not going to work, an indexer must meet all other signatures of fields declared in the same type.

Reply:
Ok, im new to typescript, and types in general. However i get compiler warnings when extending the builtin ```extenders``` object. This change made the compiler happy. A work around is to just use array like indexing like ```ko.extenders['userland'] = fn() {}```  but i was hoping to use the default dot notation.

Is there something that could be done to fix this? 

Basically the extender takes any function with a observable as a first parameter, and a list of optional parameters.

Ideas? Or should i close issue and default to use array indexing?